### PR TITLE
Documentation: Readme - Add workload philosophy section.

### PR DESCRIPTION
**Pull request by Insti**
Add a workload philosophy section to `Readme.md`

The Workload philosophy section gives some information about why an
educational workload layout is preferable to a concise one liner.

Inspired by: https://github.com/exercism/xruby/pull/567#pullrequestreview-34227224

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -218,6 +218,23 @@ class ProblemNameTest < Minitest::Test
...
+An example from `anagram`:
+```ruby
+detector = Anagram.new('allergy')
+anagrams = detector.match(["gallery", "ballerina", "regally", "clergy", "largely", "leading"])
+expected = ["gallery", "largely", "regally"]
+assert_equal expected, anagrams.sort
+```
+
+This provides the information the student needs to derive the code to pass the test in a clear and consistent manner.
+Intention revealing variable names are used to illustrate the purpose of the individual elements of what could otherwise be a one line assertion. The values used in `detector`,`anagrams` and `expected` will all vary by test.
```
Comment (hilary) [inline README.md]: "Provide the information the student needs to derive the code to pass the test in a clear and consistent manner. Illustrate the purpose of the individual elements of the assertion by using meaningful variable names."

File: README.md [hunk 2]
```diff
@@ -218,6 +218,23 @@ class ProblemNameTest < Minitest::Test
   end
 end
 ```
+### Workload philosophy.
+
+Within the test workload we want to prioritize educational value over expert comprehension and want to make sure that things are clear to people who may not be familiar with Minitest and even Ruby. 
+
+Therefore a multi line workload is preferred.
```
Comment (hilary) [inline README.md]: I would just omit this line and put the (revised paragraph) below the example here.

**Comment by hilary** [comment]
Looks great to me! 👍

**Comment by kotp** [comment]
Thank you @hilary and @Insti .

**Pull request merged**